### PR TITLE
FS-1101; Fixing fleetdb mod name

### PR DESCRIPTION
**Pull request by jakeschuurmans**
Changing go mod name from 

> sh.hollow/serverservice

 to 

> github.com/metal-toolbox/fleetdb

Bonus:
- Adding a small script to auto install crdb.

### CODE REVIEW

File: pkg/api/v1/router_firmware_set_test.go [hunk 1]
```diff
@@ -11,9 +11,9 @@ import (
 	"golang.org/x/exp/maps"
 	"golang.org/x/exp/slices"
 
-	"go.hollow.sh/serverservice/internal/dbtools"
-	"go.hollow.sh/serverservice/internal/models"
-	serverservice "go.hollow.sh/serverservice/pkg/api/v1"
+	"github.com/metal-toolbox/fleetdb/internal/dbtools"
+	"github.com/metal-toolbox/fleetdb/internal/models"
+	serverservice "github.com/metal-toolbox/fleetdb/pkg/api/v1"
```
Comment (DoctorVin) [inline pkg/api/v1/router_firmware_set_test.go:16]: I would prefer to not leave `serverservice` dangling here. Swap to `fleetdb` or the token of your choice.

File: pkg/api/v1/mock_client_test.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ import (
 	"io"
 	"net/http"
 
-	serverservice "go.hollow.sh/serverservice/pkg/api/v1"
+	serverservice "github.com/metal-toolbox/fleetdb/pkg/api/v1"
```
Comment (DoctorVin) [inline pkg/api/v1/mock_client_test.go:9]: it's only a mock but this should be `fleetdb`

File: pkg/api/v1/router_bom_test.go [hunk 3]
```diff
@@ -6,7 +6,7 @@ import (
 	"strings"
 	"testing"
 
-	serverservice "go.hollow.sh/serverservice/pkg/api/v1"
+	serverservice "github.com/metal-toolbox/fleetdb/pkg/api/v1"
```
Comment (DoctorVin) [inline pkg/api/v1/router_bom_test.go:9]: `s/serverservice/fleetdb`

File: pkg/api/v1/router_firmware_test.go [hunk 4]
```diff
@@ -9,8 +9,8 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"go.hollow.sh/serverservice/internal/dbtools"
-	serverservice "go.hollow.sh/serverservice/pkg/api/v1"
+	"github.com/metal-toolbox/fleetdb/internal/dbtools"
+	serverservice "github.com/metal-toolbox/fleetdb/pkg/api/v1"
```
Comment (DoctorVin) [inline pkg/api/v1/router_firmware_test.go:13]: I will keep noting it too. 😅

File: pkg/api/v1/router_int_test.go [hunk 5]
```diff
@@ -13,9 +13,9 @@ import (
 	"gopkg.in/square/go-jose.v2"
 	"gopkg.in/square/go-jose.v2/jwt"
 
-	"go.hollow.sh/serverservice/internal/dbtools"
-	"go.hollow.sh/serverservice/internal/httpsrv"
-	hollow "go.hollow.sh/serverservice/pkg/api/v1"
+	"github.com/metal-toolbox/fleetdb/internal/dbtools"
+	"github.com/metal-toolbox/fleetdb/internal/httpsrv"
+	hollow "github.com/metal-toolbox/fleetdb/pkg/api/v1"
```
Comment (DoctorVin) [inline pkg/api/v1/router_int_test.go:18]: `hollow` doesn't have any meaning for us. `fleetdb` or `fleetdbApi` or whatever.

File: pkg/api/v1/router_server_attributes_test.go [hunk 6]
```diff
@@ -10,8 +10,8 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"go.hollow.sh/serverservice/internal/dbtools"
-	serverservice "go.hollow.sh/serverservice/pkg/api/v1"
+	"github.com/metal-toolbox/fleetdb/internal/dbtools"
+	serverservice "github.com/metal-toolbox/fleetdb/pkg/api/v1"
```
Comment (DoctorVin) [inline pkg/api/v1/router_server_attributes_test.go:14]: serverservice delenda est!

File: pkg/api/v1/router_server_component_type_test.go [hunk 7]
```diff
@@ -8,8 +8,8 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"go.hollow.sh/serverservice/internal/dbtools"
-	serverservice "go.hollow.sh/serverservice/pkg/api/v1"
+	"github.com/metal-toolbox/fleetdb/internal/dbtools"
+	serverservice "github.com/metal-toolbox/fleetdb/pkg/api/v1"
```
Comment (DoctorVin) [inline pkg/api/v1/router_server_component_type_test.go:12]: more token replacement

File: pkg/api/v1/router_server_components_test.go [hunk 8]
```diff
@@ -10,8 +10,8 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"go.hollow.sh/serverservice/internal/dbtools"
-	serverservice "go.hollow.sh/serverservice/pkg/api/v1"
+	"github.com/metal-toolbox/fleetdb/internal/dbtools"
+	serverservice "github.com/metal-toolbox/fleetdb/pkg/api/v1"
```
Comment (DoctorVin) [inline pkg/api/v1/router_server_components_test.go:14]: serverservice token

File: pkg/api/v1/router_server_secret_types_test.go [hunk 9]
```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	serverservice "go.hollow.sh/serverservice/pkg/api/v1"
+	serverservice "github.com/metal-toolbox/fleetdb/pkg/api/v1"
```
Comment (DoctorVin) [inline pkg/api/v1/router_server_secret_types_test.go:10]: Here too.

File: pkg/api/v1/router_server_secrets_test.go [hunk 10]
```diff
@@ -9,8 +9,8 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"go.hollow.sh/serverservice/internal/dbtools"
-	serverservice "go.hollow.sh/serverservice/pkg/api/v1"
+	"github.com/metal-toolbox/fleetdb/internal/dbtools"
+	serverservice "github.com/metal-toolbox/fleetdb/pkg/api/v1"
```
Comment (DoctorVin) [inline pkg/api/v1/router_server_secrets_test.go:13]: Token replacement

File: pkg/api/v1/router_server_test.go [hunk 11]
```diff
@@ -11,8 +11,8 @@ import (
 	"github.com/stretchr/testify/require"
 	"github.com/volatiletech/null/v8"
 
-	"go.hollow.sh/serverservice/internal/dbtools"
-	serverservice "go.hollow.sh/serverservice/pkg/api/v1"
+	"github.com/metal-toolbox/fleetdb/internal/dbtools"
+	serverservice "github.com/metal-toolbox/fleetdb/pkg/api/v1"
```
Comment (DoctorVin) [inline pkg/api/v1/router_server_test.go:15]: token replacement

File: pkg/api/v1/server_component_type_service_test.go [hunk 12]
```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	hollow "go.hollow.sh/serverservice/pkg/api/v1"
+	hollow "github.com/metal-toolbox/fleetdb/pkg/api/v1"
```
Comment (DoctorVin) [inline pkg/api/v1/server_component_type_service_test.go:11]: change `hollow` to something else please

File: pkg/api/v1/server_service_test.go [hunk 13]
```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	hollow "go.hollow.sh/serverservice/pkg/api/v1"
+	hollow "github.com/metal-toolbox/fleetdb/pkg/api/v1"
```
Comment (DoctorVin) [inline pkg/api/v1/server_service_test.go:12]: I wonder if `hollow` means anything to anyone still here.

File: scripts/install_crdb.sh
Comment (DoctorVin) [inline scripts/install_crdb.sh:1]: I like the idea, but I wonder if there isn't an easier way via a package manager? I used `brew install cockroach`.

**Review by DoctorVin — CHANGES_REQUESTED** [review]
I tried to mark every place I saw a `serverservice` or a `hollow`. We should get rid of all that junk in this pass as well.

I like the "install crdb" script but I wonder if it might be better to just document it?

File: scripts/install_crdb.sh
Comment (jakeschuurmans) [reply scripts/install_crdb.sh:1]: I was simply following the instructions in the README. I keep forgetting homebrew exists. I just checked and apt and snap doesnt have it. I can rename this and remove the macOS side of it.

**Comment by jakeschuurmans** [comment]
Once this is in, [this](https://github.com/equinixmetal/k8s-fleetdb/pull/21) can be merged.

File: Makefile [hunk 14]
```diff
@@ -2,27 +2,30 @@ all: lint test
 PHONY: test coverage lint golint clean vendor local-dev-databases docker-up docker-down integration-test unit-test
 GOOS=linux
 DB_STRING=host=localhost port=26257 user=root sslmode=disable
-DEV_DB=${DB_STRING} dbname=serverservice
-TEST_DB=${DB_STRING} dbname=serverservice_test
-DOCKER_IMAGE  := "ghcr.io/metal-toolbox/fleetdb"
+DEV_DB=${DB_STRING} dbname=fleetdb
+TEST_DB=${DB_STRING} dbname=fleetdb_test
+DOCKER_IMAGE := "ghcr.io/metal-toolbox/fleetdb"
+REPO := "https://github.com/metal-toolbox/fleet-scheduler.git"
```
Comment (DoctorVin) [inline Makefile]: `fleet-scheduler`?

File: README.md [hunk 15]
```diff
@@ -1,9 +1,9 @@
-# Server Service
+# FleetDB
 
 > This repository is experimental meaning that it's based on untested ideas or techniques and not yet established or finalized or involves a radically new and innovative style!
 > This means that support is best effort (at best!) and we strongly encourage you to NOT use this in production.
```
Comment (DoctorVin) [inline README.md:4]: I only just saw that we still have this "It's untested and untrusted" thing here. We should ditch that, but if you don't want to in they PR add a Jira ticket.

**Review by DoctorVin — COMMENTED** [review]
A couple of mis-types?

File: Makefile (hunk 14 above)
Comment (jakeschuurmans) [reply Makefile]: Resolved.

File: README.md (hunk 15 above)
Comment (jakeschuurmans) [reply README.md]: Done

**Pull request closed without merging**